packages/grid/x-grid-data-generator/src/commodities.columns.tsx

```diff
@@ -32,20 +32,16 @@ import {
   randomTaxCode,
 } from './services';
 import {
-  CountryRenderer,
-  DoneRenderer,
-  EmailRenderer,
-  IncotermRenderer,
-  ProgressRenderer,
-  StatusRenderer,
+  renderCountry,
```

Comment:
I was confused by the upper case, I think they were components, turn out, they were, they were render functions. I have only realized it after trying using hooks inside, wasn't working.

Reply:
Yes it's true, the prop where they are attached is called `renderCell`